<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Settings{
border-image:url(:SettingsPage.png);
}</string>
  </property>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>374</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnBack{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUnit_price">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>204</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leUnit_price{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStock_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>284</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leStock_1{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStock_4">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>284</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leStock_4{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStock_2">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>284</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leStock_2{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePaytm_link_1">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>46</y>
     <width>589</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lePaytm_link_1{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePaytm_link_2">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>88</y>
     <width>589</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#lePaytm_link_2{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>40</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>80</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack_2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>374</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnBack_2{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>215</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn4{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>135</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn2{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>295</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn7{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>295</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn8{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>375</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn0{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>215</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn6{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDel">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>375</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnDel{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>135</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn1{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>215</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn5{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDot">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>375</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnDot{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>135</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn3{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>295</y>
     <width>69</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btn9{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leStock_3">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>284</y>
     <width>96</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#leStock_3{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTerminate">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>374</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnTerminate{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShutdown">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>374</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnShutdown{
background-color:transparent;
background:none;
border:none;
background-repeat:none;
outline:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
